slackbot/plugins/update.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+if [[ $ == 0 ]]; then
+for d in */; do
+    echo $d
+    cd $d
+    git fetch
+    git pull
+    cd ..
+done
+else
+    for d in "$@"; do
+      echo $d
+      cd $d
+      git fetch
+      git pull
+      cd ..
+    done
+fi
```

Comment:
I think we should do this in a layer outside of this repo, and keep this repo free of knowing anything about our specific implementation of updates/etc